bundles/binding/org.openhab.binding.ipx800/src/main/java/org/openhab/binding/ipx800/internal/Tuple.java

```diff
@@ -0,0 +1,16 @@
+package org.openhab.binding.ipx800.internal;
+
+/**
+ * A simple tuple implementation
+ * 
+ * @author Seebag
+ * @since 1.7.0
+ */
+public class Tuple<X, Y> { 
```

Comment:
I would prefer to give it a more domain specific name. Why not name it X to Y Mapping? Tuple doesn't seem to be used in such a generic manner hear, right?